### ISSUE TITLE
CSS Fixes - Post launch

### ISSUE THREAD

**Issue by erumaro** [erumaro/eruma-roku#4]
The following is a list of CSS fixes needed for website:

General website:
- Hamburger menu and X icon is lacking in contrast (WCAG)
- Change text align of copyright and add some vertical margin to it.
Front page on Mobile (small devices):
- The sections content are displayed outside the sections (ex. Projekt text on bg for hero).
- Maybe display featured image differently (contain?) on mobile.
Project list (small device + desktop)
- .post-type-archive-projects article, .tax-technology article {
    margin: 4em; <---- Change width
}
About page
- Jag ...  [img] and then text under. Maybe change this so image is above text?
- Let the skill text breathe more with some padding or margin. (See other texts margin/padding)
Blog
- Legibility of article footer text is really bad. Add the same color (fff) to mobile version as the desktop version.
- More horizontal space for text.

**Comment by erumaro** [comment]
Almost fixed. Some text padding still needed.

**Comment by erumaro** [comment]
Fixed.
